java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -212,6 +213,11 @@ public ProgramState apply(ProgramState input) {
     return startingStates;
   }
 
+  private static boolean parameterCanBeNull(final VariableTree variableTree) {
+    return variableTree.symbol().metadata().isAnnotatedWith("javax.annotation.CheckForNull")
```

Comment:
@DidierBesset you can assign `variableTree.symbol().metadata()` to a variable and simplify return statement.